client/me/reauth-required/index.jsx

```diff
@@ -42,7 +42,11 @@ class ReauthRequired extends Component {
 		this.props.twoStepAuthorization.off( 'change', this.update );
 	}
 
-	update = () => this.forceUpdate();
+	update = () => {
+		const reauthStatus = this.props.twoStepAuthorization.isReauthRequired();
```

Comment:
Now that we're renaming the value in the usage, I'd recommend changing this to something like `isReauthRequired`, so that it's consistent and passes a clear intention.

Reply:
> If you access [the page](https://wordpress.com/me/security/ssh-key) directly without having SSH key and authenticate, the query fails and is not retried, so the user has to manually refresh the page in order to get it working correctly.

That's correct, The issue wouldn't come up so much on that page because the common workflow is to access the profile page then security page. The profile page would show the modal and allow the reauth. 

I like option 2 above since where-ever we use `useSSHKeyQuery` it will require authorisation